cfgov/v1/jinja2/v1/newsroom/index.html

```diff
@@ -27,6 +27,11 @@
             {{ render_block.render(block, loop.index) }}
         {% endif %}
     {% endfor %}
+
+    <div class="block">
+        {% include "v1/includes/organisms/filterable-list-results.html" %}
+    </div>
```

Comment:
Interesting, so https://www.consumerfinance.gov/about-us/newsroom/ is our only layout-1-3 filterable page?

Reply:
Oh yes, right